packages/dashboard-frontend/src/services/helpers/sanitizeName.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+export default function sanitizeName(name: string): string {
+  const re = /^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$/;
+  if (re.test(name)) {
+    return name;
+  }
+
+  const sanitized = name
+    .toLowerCase()
+    .replace(/^[^a-z0-9]/, '')
+    .replace(/[^a-z0-9]$/, '')
```

Comment:
What about two or more first or last characters? It should fix but it won't. For example: 
1. ```sanitizeName(''testproject--'')``` returns ''testproject-''
2. ```sanitizeName(''--testproject'')``` returns ''-testproject''

Reply:
done